src/components/PVSearchBar.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
```

Comment:
eeeeverything from React? yikes

Reply:
It's apparently needed by react-native-elements :(

![image](https://user-images.githubusercontent.com/2262919/55692320-55109180-596c-11e9-98f7-b7a7700ff37b.png)